### ISSUE TITLE
Are all default data fields signed?

### ISSUE THREAD

**Issue by nayuta-ueno** [lightningnetwork/lightning-rfc#385]
> BOLT#1 : `All data fields are big-endian`

Are all default data fields signed?

ex:
  * `[2:gflen]` --> 0～32767(signed) or 0～65535(unsigned)

**Comment by practicalswift** [comment]
No, they are unsigned by default. AFAIK there is not a single field in the spec that is signed.

**Comment by nayuta-ueno** [comment]
Thank you.
I wrote PR.
